src/Diagram.js

```diff
@@ -368,8 +375,9 @@ class Diagram extends React.PureComponent {
   }
 
   renderChildren(extremeX, extremeY, zoom) {
-    const verticesMap = this.getVisibleVertices(zoom);
-    const edges = this.getVisibleEdges(zoom);
+    const overscan = 1;
```

Comment:
lets take a prop `overscanPixels` as a number. and fallback of that should be width and height of the container

Reply:
done